tools/changelogger-release.sh

```diff
@@ -171,6 +171,19 @@ for SLUG in "${TO_RELEASE[@]}"; do
 	info "Processing $SLUG..."
 	RELEASED[$SLUG]=1
 
+	# Avoid "There are no changes with content for this write. Proceed?" prompts and empty changelog entries.
+	ANY=false
+	for f in "$CHANGES_DIR"/*; do
+		if [[ -n $( sed -n -e '/^$/,$ { /[^ \t]/ p }' "$f" ) ]]; then
```

Comment:
```suggestion
		if [[ -n $( sed -n -e '/^$/,$ { /[^ \t]/ p; }' "$f" ) ]]; then
```

So I'm still getting the error, and after doing some more testing, it seems like adding a semicolon here is the only thing that fixes it for some reason?